packages/stems/src/components/SelectablePill/SelectablePill.tsx

```diff
@@ -0,0 +1,44 @@
+import { forwardRef } from 'react'
+
+import cn from 'classnames'
+
+import styles from './SelectablePill.module.css'
+import { SelectablePillProps } from './types'
+
+export const SelectablePill = forwardRef<
+  HTMLButtonElement,
+  SelectablePillProps
+>(
+  ({
```

Comment:
personally can we just call this props, so its `props: SelecteablePillProps` and then destructure in the body? i think it makes the layout a bit cleaner and consistent